en/_posts/2016-04-20-marathon-health-checks.md

```diff
@@ -6,23 +6,23 @@ author: devatoria
 lang: en
 ---
 
-Health checks are useful to ensure that your container is running; and better, is running as expected. You can add multiple health checks per application. A health check can be of two types:
+Health checks are useful to ensure that your containers are running; and even better, are running as expected. You can add multiple health checks per application. A health check can be of two types:
```

Comment:
- (non blocking suggestion): are behaving as expected
- (non blocking suggestion): Marathon supports both high level HTTP and lower level TCP health checks